docker/env.js

```diff
@@ -0,0 +1,7 @@
+window.$env = {
+  // Choose one of the below,
+  // a) localhost, internal deployments
+  RECAPTCHA_SITEKEY: "6LcH8qYbAAAAAGu7TrMGZTMYanTY7HIs8b6VIdj4",
+  // b) public deployment
+  // RECAPTCHA_SITEKEY: "6Les9qYbAAAAAEge76hhNGUyBnEwjxdwGCB8zmQm",
+};
```

Comment:
What are these keys?
Shouldn't the user set them? If yes, we should document it in the docs that the user should configure this file